### PR TITLE
test: Fix choice chips screenshot test

### PR DESCRIPTION
**Pull request by patrickrodee**

### CODE REVIEW

**Comment by abhiomkar** [comment]
@moog16 is fixing the flaky test related to checkbox ripple in this PR #4735 so we don't have to update the goldens here.

**Comment by patrickrodee** [comment]
Blocked by #4735

**Comment by patrickrodee** [comment]
@abhiomkar @moog16 The checkbox Firefox screenshot was still failing so I've updated it here.

File: test/screenshot/golden.json [hunk 1]
```diff
@@ -208,10 +208,10 @@
     }
   },
   "spec/mdc-checkbox/mixins/touch-dimension.html": {
-    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/18_29_09_183/spec/mdc-checkbox/mixins/touch-dimension.html?utm_source=golden_json",
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/23_37_03_603/spec/mdc-checkbox/mixins/touch-dimension.html?utm_source=golden_json",
```
Comment (moog16) [inline test/screenshot/golden.json]: I don't think you have the latest changes from develop. I moved the "autofocus" to the unchecked, checkbox. It should look like https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/18_29_09_183/spec/mdc-checkbox/classes/baseline.html?utm_source=golden_json. Changed in https://github.com/material-components/material-components-web/pull/4735/files#diff-c6902490070f225884188887b3d3feecR49

Comment (patrickrodee) [reply test/screenshot/golden.json]: Git shows I'm up to date with develop though.

**Comment by patrickrodee** [comment]
@moog16 @abhiomkar I'm up-to-date with develop and have passing screenshot tests after adding the previously failing Firefox checkbox screenshot. If I revert the approved Firefox checkbox screenshot update, the screenshot tests fail. Let me know how you'd like me to proceed. I do not believe the changes in #4735 resolved the same issue I'm seeing here.

File: test/screenshot/golden.json (hunk 1 above)
Comment (moog16) [reply test/screenshot/golden.json]: Sorry my bad...this is under `mixins`. I only updated it in the `classes/baseline.html` file.

**Comment by moog16** [comment]
@patrickrodee its all fixed!

**Comment by patrickrodee** [comment]
@moog16 awesome, thanks for the help! want to stamp this, too? :D

**Pull request merged**